src/commands/scanner/run.ts

```diff
@@ -110,6 +79,14 @@ export default class Run extends ScannerCommand {
 			description: messages.getMessage('flags.tsconfigDescription'),
 			longDescription: messages.getMessage('flags.tsconfigDescriptionLong')
 		}),
+		eslintconfig: flags.string({
```

Comment:
New flags - `eslintconfig`, `pmdconfig` - please let me know if you have suggestions for more concise names.

Reply:
I like them. I don't think we should add hyphens. But in case someone else does, it is possible to add hyphens if the argument is specified as a string. i.e. `'eslint-config'`.

Reply:
I agree - Salesforce CLI doc review team usually asks us to not use hyphens in flags. Will leave it as is then.